ui/components/app/confirm/info/row/address.tsx

```diff
@@ -62,7 +63,7 @@ export const ConfirmInfoRowAddress = memo(
                   color={TextColor.inherit}
                   data-testid="confirm-info-row-display-name"
                 >
-                  {displayName}
+                  {isSnapUsingThis ? shortenAddress(address) : displayName}
```

Comment:
`useFallbackDisplayName` that returns display name includes shortenAddress but also other potential sources of names such as ENS or the wallet address book. Why do we want to ignore those?

Reply:
The display name returned by `useFallbackDisplayName` is tied to the currently selected network in the UI. This causes confusion when Snaps may be showing addresses for chains that are not the currently selected network.

Generally the Snaps team would like to use all of this functionality, including petnames, but not before it has been decoupled from the currently selected network.